src/jdk.httpserver/share/classes/sun/net/httpserver/ServerConfig.java

```diff
@@ -55,91 +52,71 @@ class ServerConfig {
     private static long idleTimerScheduleMillis;
     private static long idleIntervalMillis;
```

Comment:
while you're at it you could make these two fields final as well and use temporary variables in the static block to make sure they're assigned only once.

Reply:
Done.